packages/candy_editor/src/constants.ts

```diff
@@ -1 +1,19 @@
-export const NOT_SELECTED = 'Not selected';
\ No newline at end of file
+export const NOT_SELECTED = 'Not selected';
+export const SELECT_OPTIONS = [
+  { label: 'Text', value: 'Text' },
+  { label: 'Nat', value: 'Nat' },
+  {  label: 'Nat8', value: 'Nat8'},
```

Comment:
Not formatted with prettier. Please configure VS Code "formatter" and "On Save" settings to automatically format when saving a file.